cmd/tools/tools.go

```diff
@@ -61,7 +61,7 @@ func init() {
 func Run(cmd *cobra.Command, args []string) {
 	_, _ = cmd, args
 
-	provider := boot.NewProvider(importBootConfPath)
+	provider := boot.NewDiscovery(importBootConfPath)
```

Comment:
It might be better to change provider to discovery

Reply:
It's a bit weird that `NewProvider` returns a `Discover` interface, that's why I renamed it, but you're right, no need to change it, I'll recovery it, thanks.

Reply:
👍